src/integrationtest/java/com/ericsson/gerrit/plugins/eiffel/integrationtest/ServiceIntegrationSteps.java

```diff
@@ -0,0 +1,282 @@
+        generateBody.addProperty("submit_type", "INHERIT");
+        generateBody.addProperty("create_empty_commit", true);
+
+        HttpRequest request = new HttpRequest(HttpMethod.PUT);
+        ResponseEntity response;
+        response = request.setHeader("Cookie", gerritAccountCookie)
+                .setHeader("Content-Type", "application/json; charset=UTF-8")
+                .setHeader("Accept", "application/json")
+                .setHeader("X-Gerrit-Auth", gerritXAuthToken).setBaseUrl(GERRIT_BASE_URL)
+                .setEndpoint("/projects/" + projectName).setBody(generateBody.toString())
+                .performRequest();
+
+        return response;
+    }
+
+    private ResponseEntity enableEiffelMessaging(String publishUrl)
+            throws URISyntaxException, ClientProtocolException, IOException {
+        String jsonBodyString = "{\"description\":\"Auto generated project\","
+                + "\"use_contributor_agreements\":\"INHERIT\",\"use_content_merge\":\"INHERIT\","
+                + "\"use_signed_off_by\":\"INHERIT\",\"require_change_id\":\"INHERIT\","
+                + "\"create_new_change_for_all_not_in_target\":\"INHERIT\","
+                + "\"reject_implicit_merges\":\"INHERIT\",\"submit_type\":\"MERGE_IF_NECESSARY\","
+                + "\"state\":\"ACTIVE\",\"plugin_config_values\":{\"Eiffel-Integration\":"
+                + "{\"enabled\":{\"value\":\"true\"},\"filter\":{},\"flow-context\":{},"
+                + "\"remrem-password\":{},\"remrem-publish-url\":{\"value\":\"" + publishUrl
+                + "\"},"
+                + "\"remrem-username\":{}},\"uploadvalidator\":{\"binaryTypes\":{\"values\":[]},"
+                + "\"blockedContentType\":{\"values\":[]},\"blockedContentTypeWhitelist\":"
+                + "{\"value\":\"false\"},\"blockedFileExtension\":{\"values\":[]},"
+                + "\"blockedKeywordPattern\":{\"values\":[]},\"invalidFilenamePattern\":"
+                + "{\"values\":[]},\"maxPathLength\":{\"value\":\"0\"},\"project\":"
+                + "{\"values\":[]},\"ref\":{\"values\":[]},\"rejectDuplicatePathnames\":"
+                + "{\"value\":\"false\"},\"rejectDuplicatePathnamesLocale\":{\"value\":\"en\"},"
+                + "\"rejectSubmodule\":{\"value\":\"false\"},\"rejectSymlink\":"
+                + "{\"value\":\"false\"},\"rejectWindowsLineEndings\":{\"value\":\"false\"},"
+                + "\"requiredFooter\":{\"values\":[]}}}}";
+
+        HttpRequest request = new HttpRequest(HttpMethod.PUT);
+        ResponseEntity response;
+        response = request.setHeader("Cookie", gerritAccountCookie)
+                .setHeader("Content-Type", "application/json; charset=UTF-8")
+                .setHeader("Accept", "application/json")
+                .setHeader("X-Gerrit-Auth", gerritXAuthToken).setBaseUrl(GERRIT_BASE_URL)
+                .setEndpoint("/projects/" + projectName + "/config").setBody(jsonBodyString)
+                .performRequest();
```

Comment:
Why type out this in every method? Wouldn't it be better to put it in a method?

Reply:
done in next commit